### DISCUSSION TITLE
Where is the 28.1.2 32 bits?

### DISCUSSION THREAD

**Discussion by jozephbrasil** [obsproject/obs-studio#7817] [Development Questions] [answered]
Anyone knows where may I download the version 28.1.2 for 32 bits platform?

**Comment by derrod** [comment, marked as answer]
There are no 32 bit builds of OBS 28 or later.

**Comment by Mister36MN** [comment]
As of OBS 28, there is no 32-bit version of OBS. It is highly suggested to update your OS to a version that supports 64-bit applications.

Please use discord or the forums for support questions in the future.
